web/src/main/java/org/phoenixctms/ctsms/web/jersey/resource/proband/ProbandResource.java

```diff
@@ -365,32 +344,6 @@ public ProbandOutVO updateProband(ProbandInVO in) throws AuthenticationException
 				Settings.getIntNullable(SettingCodes.API_GRAPH_MAX_PROBAND_CHILDREN_DEPTH, Bundle.SETTINGS, DefaultSettings.API_GRAPH_MAX_PROBAND_CHILDREN_DEPTH));
 	}
 
-	// @PATCH
```

Comment:
why not trying to bring PATCH to life?

Reply:
see jersey upgrade path

Reply:
ack. but afair jersey2 means rewriting it (not too huge though).